CesiumGeospatial/CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ PUBLIC
     earcut
 PRIVATE
     s2::s2
+    absl::string_view
```

Comment:
Since `absl::string_view` is used from the `S2CellID` class, I'd assumed it was an omission. However, looking more closely, `absl::string_view` comes as a dependency of `s2` itself, so this is unnecessary.